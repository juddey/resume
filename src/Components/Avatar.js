import React from 'react'
import styles from './Styles/AvatarStyle'
import MuiAvatar from '@material-ui/core/Avatar'
import { Images } from '../Themes'

export default class Avatar extends React.Component {
  render () {
    return (
      <div style={styles.container}>
        <MuiAvatar
          alt='juddey'
          src={Images.avatar}
          style={{ width: 60, height: 60 }}
        />
      </div>
    )
  }
}
