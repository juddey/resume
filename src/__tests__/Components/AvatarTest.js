import React from 'react'
import {
  createComponentWithIntl,
  shallowWithIntl
} from '../../../src/Support/Utils'
import Avatar from '../../Components/Avatar'

const div = document.createElement('div')
const wrapper = shallowWithIntl(<Avatar />)

test('component exists', () => {
  expect(wrapper.length).toBe(1)
})

test('component structure', () => {
  expect(wrapper.name()).toEqual('div')
  expect(wrapper.children().length).toEqual(1)
})

test('renders to snapshot', () => {
  const tree = createComponentWithIntl(<Avatar />, div).toJSON()
  expect(tree).toMatchSnapshot()
})
