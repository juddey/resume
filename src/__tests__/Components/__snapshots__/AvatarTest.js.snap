// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to snapshot 1`] = `
<div
  style={
    Object {
      "flex": 1,
    }
  }
>
  <div
    className="MuiAvatar-root-1"
    style={
      Object {
        "height": 60,
        "width": 60,
      }
    }
  >
    <img
      alt="juddey"
      className="MuiAvatar-img-3"
      sizes={undefined}
      src="avatar.jpg"
      srcSet={undefined}
    />
  </div>
</div>
`;
